### ISSUE TITLE
Tiny mouse in GUI apps when enabling fractional scaling on GNOME Wayland

### ISSUE THREAD

**Issue by kreaninw** [89luca89/distrobox#745]
Please, before opening a bug:

- make sure you've read the documentation.
- Ensure there isn't already an open issue about this.
- Ensure there isn't already a closed/resolved issue about this.

**Describe the bug**
The mouse cursor is tiny in all GUI apps running on GNOME Wayland with fractional scaling enabled, 125% in my case, as shown in the screenshot below:

![Screenshot from 2023-05-09 04-07-19](https://user-images.githubusercontent.com/1403194/237035190-c3316008-daca-4866-9877-bf1265c7556e.png)

**To Reproduce**
1. `distrobox create -n ubuntu -i ubuntu:22.04`
2. `distrobox enter ubuntu`
3. `sudo apt update`
4. `sudo apt  full-upgrade`
5. `sudo apt install menulibre` (for testing)
6. `menulibre` >>> The mouse cursor inside the app window is tiny.

**Expected behavior**
The mouse cursor should be at the normal size.

**Logs**
-

**Desktop (please complete the following information):**
 - Are you using podman or docker? I am using a docker.
 - Which version or podman or docker? 23.0.5_ce-1.1
 - Which version of distrobox? 1.4.2.1-3.1

**Additional context**

**Comment by rainwashed** [comment]
can vouch that this also occurs with KDE but on x11

**Comment by mcdorians** [comment]
For me the cursor in all GTK apps was the right scale with the host scale at 200%. But the QT was at 100% and tiny. I tried to scale the cursor in a lot of ways the QT specific didn't work and the X11 and wayland ones scaled also the host. So i did a dirty hack by giving the distrobox container a different cursor theme.

1. Download a cursor with hdpi. eg. https://www.gnome-look.org/p/1148692
2.  extract and move to icons. Importst use the system path not the home at this is shared with the host. `sudo mv Downloads/capitaine-cursors-r4/capitaine-cursors/ /usr/share/icons/`
3. Set the name of the new cursor theme. `sudo nano /usr/share/icons/default/index.theme `
4. `sudo update-alternatives --config x-cursor-theme`
5. restart the container

by the way this tiny cursor problem only occurred in my ubuntu 22.04 containers. debian ones are fine.

**Comment by archerallstars** [comment]
Thanks to https://github.com/89luca89/distrobox/issues/1026#issuecomment-2001737552, I can work around with this issue myself. The fix would depend on the container image. For example, to fix this issue in Arch container, all I need to do is:

```
sudo pacman -Syu
sudo pacman -S adwaita-cursors
```
_* See: https://archlinux.org/packages/extra/any/adwaita-cursors/_

I didn't test with Ubuntu image, but it probably is the same:

```
sudo apt update
sudo apt adwaita-icon-theme
```
_* See: https://launchpad.net/ubuntu/+source/adwaita-icon-theme_

Or you might want to install [`yaru-theme`](https://launchpad.net/ubuntu/+source/yaru-theme) instead for Ubuntu host.

Basically, this issue cause by the lacking of the cursor theme in the container. I don't know what's the proper way to fix this issue from Distrobox side. Feel free to close this issue, as it's a container's image issue. In this case, we would need a container setup doc referring to this issue.

**Comment by AuraHerreroRuiz** [comment]
> Basically, this issue happens because of the lacking of the cursor theme in the container image. I don't know what's the proper way to fix this issue from Distrobox side. Feel free to close this issue, as it's an image issue. In this case, we would need a container setup doc to prevent this issue.
Another way I've found can be used to fix this issue is to mount the directory of the cursor theme currently used as a read-only volume (specially useful if it isn't available on the image's distribution package manager):
`--volume /usr/share/icons/cursor_theme_directory:/usr/share/icons/cursor_theme_directory:ro`
or, in a distrobox.ini 
`volume="/usr/share/icons/cursor_theme_directory:/usr/share/icons/cursor_theme_directory:ro"`

**Comment by archerallstars** [comment]
@AuraHerreroRuiz 

In my journey to install Brave in a Distrobox container, I run in to an issue when mounting `--volume /usr/share/icons:/usr/share/icons`, even when I mount with `rw`. Basically, any app that tries to install dependencies related to theming will fail in a rootless container.

I'm testing this in a rootless Podman container on a Tumbleweed host.

So, I would recommend that anyone who running into this issue install the theme package instead. The package is [`adwaita-icon-theme`](https://build.opensuse.org/package/show/openSUSE:Factory/adwaita-icon-theme) in Tumbleweed container, by the way.
